site/profile/facts.d/terraform_facts.yaml

```diff
@@ -0,0 +1,2 @@
+---
+stack: "computecanada"
```

Comment:
If this file can be generated from `terraform_facts.yaml.tmpl` then this should be good to go

Reply:
Done in https://github.com/ComputeCanada/magic_castle/pull/124